README.md

```diff
@@ -41,47 +47,48 @@ extend the core metadata capabilities of Strawberryfield and allow the same flex
+In its guts (or heart?), Archipelago does something quite simple but core to our concept of repository: it transforms in realtime the _close to your needs open schema metadata_ that lives in strawberryfield as JSON into _close to other one's fixed schema needs metadata_; any destination format, using a fast, cached templating system. A templating system easy enough to master and core to Drupal, called [`Twig`](https://twig.symfony.com/doc/2.x/).
 
-This templating system is exposed to Archipelago users through the UI and stored side by side in the repository as content 
+This templating system is exposed to Archipelago users through the UI and stored side by side in the repository as content
 (we named them `Metadata Display entities`, but they not only serve display needs!) so users can fully control how metadata is transformed and published without touching their individual sources.
 
 Templates or recipes can be shared, exported, ingested, updated, and adapted in many ways. Fast changes are possible without having to wait for the next mayor release of Archipelago or your favorited Metadata Schema Specs Committee agreeing on the next or the last version. Of course, this module not only handles metadata but media assets too, extracting local or remote URIs and files from your metadata and rendering them as media viewers: books, 3D models, images, panoramas, A/V with IIIF in its soul. Discover what [`format_strawberryfield can`](format.md) do and what many other possibilities are exposed through our templating system.
 
-# Archipelago Deployment Quickstart
-
-To get a general glance of what is included and how it works deploy our latest beta locally using `Docker` using [this Guide](https://github.com/esmero/archipelago-deployment/blob/8.x-1.0-beta1/README.md) covering for now OSX and Ubuntu 18.04.
+# Code of Conduct
+The Archipelago Commons community is dedicated to providing a welcoming and positive experience for all participants. To ensure this can be met for all parties, please review our [Code of Conduct](docs/CODE_OF_CONDUCT.md).
 
-# Archipelago Usage Quickstart
+# Archipelago Deployment Quickstart
+To get a general glance of what is included and how it works, follow [this guide](https://github.com/esmero/archipelago-deployment/blob/8.x-1.0-beta1/README.md) to deploy our latest beta locally using `Docker` with OSX or Ubuntu 18.04.
 
-Now that you have it running, it's time to ingest your first digital object. Follow this [Guide](docs/firstobject.md) to master the basics.
+# Archipelago Usage Guides
+Now that you have Archipelago running, the following guides will help you master the basics.
+* [Ingesting Your First Object](docs/firstobject.md)
+* [Creating Display Modes](docs/createdisplaymodes.md)
+* [How to Create a Webform as an Input Method for ADO](docs/webformsasinput.md)
```

Comment:
I know we discussed yesterday and I'm down for ADO. But here on the welcome page, it's not been defined yet? So take or leave this comment.
```suggestion
* [How to Create a Webform as an Input Method for Digital Object](docs/webformsasinput.md)
```

Reply:
ADO is an acronym for Archipelago Digital Object, so either its directly `ADO` or `Archipelago Digital Object (ADO)`

Reply:
@DiegoPino updated. Thanks.

Reply:
@marlo-longley Thnks! I have NO permission to push changes on the @mitchellkeaney fork, so he will have to do it, or i merge as it is and the make an amend.

Reply:
@mitchellkeaney can you *please* commit the suggestions so we can move forward with this? Thanks!